package.json

```diff
@@ -52,12 +52,12 @@
   },
   "version": "13.3.0-next.0",
   "dependencies": {
-    "@angular/animations": "^13.2.0-next.2",
-    "@angular/common": "^13.2.0-next.2",
-    "@angular/compiler": "^13.2.0-next.2",
-    "@angular/core": "^13.2.0-next.2",
-    "@angular/forms": "^13.2.0-next.2",
-    "@angular/platform-browser": "^13.2.0-next.2",
+    "@angular/animations": "13.2.0-rc.1",
```

Comment:
FYI: Removing the caret here as it's better pinning here too, making it obvious which Angular version we use (otherwise the lock file might accidentally be updated without the package.json reflecting the current version)

Reply:
I put the caret back in last time I changed the version, because it's closer to how people usually install Angular in their own apps. I don't think it a big deal for the lock file to point to a different patch version than the one we're using since we're meant to be compatible with it anyway.

Reply:
Yes, definitely shouldn't cause any issues. Happy to re-add it but I guess it doesn't matter a lot, and FW doesn't do it either for its cross-Angular dependencies.

Reply:
Up to you, I don't feel strongly about it.

Reply:
sgtm. feel free to re-add caret when we update next time, or leave it as is 😛